plugins/woocommerce/src/Admin/Features/BlockEditorFeatureEnabled.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * WooCommerce Block Editor Feature Endabled
+ */
+
+namespace Automattic\WooCommerce\Admin\Features;
+
+use Automattic\WooCommerce\Admin\Features\TransientNotices;
+use Automattic\WooCommerce\Admin\PageController;
+use Automattic\WooCommerce\Internal\Admin\Loader;
+
+/**
+ * Loads assets related to the new product management experience page.
+ */
+class BlockEditorFeatureEnabled {
+
+	const FEATURE_ID = 'block-editor-feature-enabled';
+
+	/**
+	 * Option name used to toggle this feature.
+	 */
+	const TOGGLE_OPTION_NAME = 'woocommerce_' . self::FEATURE_ID . '_enabled';
+
+	/**
+	 * Constructor
+	 */
+	public function __construct() {
+		if ( ! Features::is_enabled( 'new-product-management-experience' ) && Features::is_enabled( self::FEATURE_ID ) ) {
```

Comment:
So, if both feature flags are enabled, the block editor is still displayed but this condition is not met, so the styles/etc are not enqueued. Should this just be:
```php
if (Features::is_enabled( self::FEATURE_ID ) )
```

?

Reply:
Ah, understood!